docs/src/_getting-started/part3.md

```diff
@@ -0,0 +1,176 @@
+[...]
+"sbalnojan/meltano-example-el"}, "time_extracted": "2022-09-22T07:37:06.289545Z"}
+
+...[many more records]...
+
+{"type": "STATE", "value": {"bookmarks": {"sbalnojan/meltano-example-el": {"commits": {"since": "2022-09-22T07:37:06.289545Z"}}}}}
+´´´
+</div>
+
+## Installing the transform-field mapper
+To add inline transformations, we need a new plugin. We're going to use the mapper "transform-field". To add this plugin, use the `meltano add mapper` command:
+
+<div class="termy">
+```console
+$ meltano add mapper transform-field
+
+Added mapper 'transform-field' to your Meltano project
+Variant:        transferwise (default)
+Repository:     https://github.com/transferwise/pipelinewise-transform-field
+Documentation:  https://hub.meltano.com/mappers/transform-field
+
+Installing mapper 'transform-field'...
+---> 100%
+Installed mapper 'transform-field'
+
+To learn more about mapper 'transform-field', visit https://hub.meltano.com/mappers/transform-field
+</div>
+
+We're now going to add two mappings to this mapper.
+
+## Adding a emails-hidden mapping
+To add our first mapping, we're going to edit the `meltano.yml` file located inside your root project directory. Modify the block for the `transform-field` mapper as shown below:
+
+```yaml
+mappers:
+  - name: transform-field
+    variant: transferwise
+    pip_url: pipelinewise-transform-field
+    executable: transform-field
+   mappings:
+    - name: hide-github-mails
+      config:
+        transformations:
+          - field_id: "commit"
+            tap_stream_name: "commits"
+            field_paths: ["author/email", "committer/email"]
+            type: "MASK-HIDDEN"
+```
+Let's go through this step-by-step
+
+```yaml
+mappers:
+  [...]
+   mappings:
+    - name: hide-github-mails
+      config:
+        transformations:
+          [...]
+```
+These lines define the name "hide-github-mails" as the name of our mapping. We can call the mapping using this name, and ignoring any reference to the actual mapper "transform-field".
+
+```yaml
+    [...]
+        transformations:
+          - field_id: "commit"
+            tap_stream_name: "commits"
+            field_paths: ["author/email", "committer/email"]
+            type: "MASK-HIDDEN"
+```
+These lines define one transformation. We instruct to target the stream "commits", and therein the field "commit". We then use the field paths to navigate to the two emails we know are contained within this message and set the type to "MASK-HIDDEN" to hide the values.
+
+## Run the data integration (E(t)L) pipeline
+Now we're ready to run the data integration process with these modifications again. To do so, we'll need to clean up first, since we already ran the EL process in part 1. The primary key is still the same and as such the ingestion would fail.
+
+Drop the table inside your local postgres by running
+
+```sql
+DROP TABLE tap_github.commits
```

Comment:
I don't know, had the same reaction, different schema isn't cool because we want to do it again in part 4. I could wrap it into a docker command, after all they need to have docker anyways.